### DISCUSSION TITLE
About usage of patroni_standby_cluster.

### DISCUSSION THREAD

**Discussion by kazimgun** [autobase-tech/autobase#797] [Q&A]
I want to create two zone Patroni Cluster, with spereate etcd and haproxy cluster.  In playbook there is some settings for patroni_standby_cluster but I couldn't realize how can I mange this. Could you explain methodology, for example first deploy standby cluster with this settings, and than deploy primary zone or deploy two zone with defining variables like this?

**Comment by vitabaks** [comment]
Hello. I will describe the details in the documentation soon.

**Comment by kazimgun** [comment]
Hi,
I resolved my issue by using multiple vars/main.yml files and running the playbook with the -e option to specify which main file should be used.

For documentation: First, I deployed the primary cluster. Then, to set up the standby cluster, I edited vars/main_standby.yml—a copy of the original main.yml file—and configured patroni_standby_cluster with the IP addresses from the primary cluster."

**Reply by vitabaks** [reply]
Yes, that's right. I will try to describe it in more detail in the documentation.
